librz/util/table.c

```diff
@@ -1179,7 +1212,39 @@ RZ_API void rz_table_visual_list(RzTable *table, RzList *list, ut64 seek, ut64 l
 		}
 	}
 }
+RZ_API RzTable *rz_table_transpose(RzTable *t) {
+	RzListIter *iter;
+	RzListIter *iter1;
+	RzListIter *iter2;
+	RzTableRow *row;
+	RzList *row_name = rz_list_new();
+	RzTableColumn *col;
+	RzStrBuf *tmp = rz_strbuf_new("");
+	RzTable *transpose = rz_table_new();
+	RzTableColumnType *typeString = rz_table_type("string");
 
+	rz_table_add_column(transpose, typeString, "KEY", 0);
+	rz_table_add_column(transpose, typeString, "VALUE", 0);
+	//printf("total coulumns\n%d", t->totalRows);
```

Comment:
Remove commented code

Reply:
Could you also suggest what I could use for the column name instead of KEY and VALUE.

Reply:
Probably "NAME" is better than "KEY"